repository-data/application/src/main/resources/hcm-config/hst/configurations/common/pages/static-ui-news-hub.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
I still need to understand why there are static ui configuration, but this is not relevant for this review